sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubsTestEnvironment.cs

```diff
@@ -29,6 +29,15 @@ public sealed class EventHubsTestEnvironment: TestEnvironment
         /// <summary>The name of the environment variable used to specify an override for the Event Hub instance to use for all tests.</summary>
         private const string EventHubNameOverrideEnvironmentVariable = "EVENTHUB_OVERRIDE_EVENT_HUB_NAME";
 
+        /// <summary>The name of the environment variable used to specify azure authority host to use for all tests.</summary>
+        private const string AzureAuthorityHostEnvironmentVariable = "AZURE_AUTHORITY_HOST";
+
+        /// <summary>The name of the environment variable used to specify service management url to use for all tests.</summary>
+        private const string ServiceManagementUrlEnvironmentVariable = "SERVICE_MANAGEMENT_URL";
+
+        /// <summary>The name of the environment variable used to specify resource manager to use for all tests.</summary>
+        private const string ResourceManagerEnvironmentVariable = "RESOURCE_MANAGER";
```

Comment:
This should be `RESOURCE_MANAGER_URL` to map nicely with `Get-AzEnvironment` output.